web-common/src/features/feature-flags.ts

```diff
@@ -32,6 +32,14 @@ class FeatureFlags {
   readOnly = new FeatureFlag("rill", false);
 
   ai = new FeatureFlag("user", !import.meta.env.VITE_PLAYWRIGHT_TEST);
+  /**
+   * We make some calls to cloud for fetching user related stats. But this is not needed in playwring tests.
+   * While there wont be token in CI there could be one when running on a dev's laptop.
+   */
+  disableCloud = new FeatureFlag(
```

Comment:
We should be clear on the difference between "disableCloud" and "adminServer" (which also is a toggle for Cloud features). It seems the difference is whether or not we show Cloud features _in Rill Developer_, so I'd propose something like: `rillDevCloudFeatures`, `rillDevAdminServer`, or something along those lines. 

Also, I prefer the feature flags are positive not negative. It's easier to read `if (feature)` rather than `if (!noFeature)`.